src/sentry/api/base.py

```diff
@@ -234,6 +234,10 @@ def dispatch(self, request, *args, **kwargs):
                     # setup default access
                     request.access = access.from_request(request)
 
+            if hasattr(self.request.successful_authenticator, "token_name"):
```

Comment:
```python
if getattr(self.request.successful_authenticator, "token_name", None) == "bearer":
```
Also should this be case-sensitive?

Reply:
The token name is set here: https://github.com/getsentry/sentry/blob/master/src/sentry/api/authentication.py#L184 so it should be alright.

Reply:
I updated it to directly check against `TokenAuthentication.token_name`

Reply:
```python
if getattr(self.request.successful_authenticator, "token_name", None) == TokenAuthentication.token_name:
  request._metric_tags["backend_request"] = True
```

Reply:
Whoops, didn't see the `..., None)` initially